library/core/src/iter/traits/collect.rs

```diff
@@ -494,7 +494,7 @@ impl Extend<()> for () {
 
 macro_rules! spec_tuple_impl {
     ( ($ty_name:ident, $var_name:ident, $extend_ty_name: ident, $trait_name:ident, $default_fn_name:ident, $cnt:tt), ) => {
-        spec_tuple_impl!($trait_name, $default_fn_name, #[doc(fake_variadic)] #[doc = "This trait is implemented for tuples up to twelve items long. The `impl`s for 1- and 3- through 12-ary tuples were stabilized after 2-tuples, in RUSTC_CURRENT_VERSION."] => ($ty_name, $var_name, $extend_ty_name, $cnt),);
+        spec_tuple_impl!($trait_name, $default_fn_name, #[doc(fake_variadic)] #[doc = "This trait is implemented for tuples up to twelve items long. The `impl`s for 1- and 3- through 12-ary tuples were stabilized after 2-tuples, in 1.85.0."] => ($ty_name, $var_name, $extend_ty_name, $cnt),);
```

Comment:
While you're here might as well wrap these lines to 100 characters, this macro is kind of hard to read. There are a few places below that should get updated too.

```rust
macro_rules! spec_tuple_impl {
    (
        (
            $ty_name:ident, $var_name:ident, $extend_ty_name: ident,
            $trait_name:ident, $default_fn_name:ident, $cnt:tt
        ),
    ) => {
        spec_tuple_impl!(
            $trait_name,
            $default_fn_name,
            #[doc(fake_variadic)]
            #[doc = "This trait is implemented for tuples up to twelve items long. The `impl`s for \
                     1- and 3- through 12-ary tuples were stabilized after 2-tuples, in 1.85.0."] 
            => ($ty_name, $var_name, $extend_ty_name, $cnt),
        );
    };
    // ...
```

Reply:
Thank you, and in particular for the hints about what it should look like